gradle.properties

```diff
@@ -2,22 +2,20 @@ KOTLIN_VERSION=1.4.20
 BINTRAY_VERSION=1.8.3
 ARTIFACTORY_VERSION=4.17.2
 KTLINT_VERSION=9.2.1
-
 JACOCO_VERSION=0.8.3
-
 KOTLIN_COROUTINES_VERSION=1.4.2
 SL4J_VERSION=1.7.25
 JODA_VERSION=2.9.7
 JODA_CONVERT_VERSION=1.8.1
 NETTY_VERSION=4.1.49.Final
 KOTLIN_LOGGING_VERSION=1.7.6
+SCRAM_CLIENT_VERSION=1.0.0-beta.2
```

Comment:
Even though this is a beta version, it's the [version used by the official PostgreSQL JDBC driver](https://github.com/ahachete/pgjdbc/blob/f3ade07d7758f134fd137db3ebab544f2211e290/pgjdbc/pom.xml#L43) and it's the latest version that exists.

Reply:
Wow, that's embarrassing. I was looking at the fork instead of the official JDBC driver. Very good catch!